dnf-behave-tests/features/command-aliases.feature

```diff
@@ -0,0 +1,104 @@
+        | provides            | whatprovides                 |
+        | provides            | prov                         |
+        | reinstall           | reinstall                    |
+        | reinstall           | rei                          |
+        | remove              | remove                       |
+        | remove              | erase                        |
+        | remove              | rm                           |
+        | remove              | remove-n                     |
+        | remove              | remove-na                    |
+        | remove              | remove-nevra                 |
+        | remove              | erase-n                      |
+        | remove              | erase-na                     |
+        | remove              | erase-nevra                  |
+        | repolist            | repolist                     |
+        | repolist            | repoinfo                     |
+        | repoquery           | rq                           |
+        | repoquery           | repoquery                    |
+        | repoquery           | repoquery-n                  |
+        | repoquery           | repoquery-na                 |
+        | repoquery           | repoquery-nevra              |
+        | repository-packages | repository-packages          |
+        | repository-packages | repo-pkgs                    |
+        | repository-packages | repo-packages                |
+        | repository-packages | repository-pkgs              |
+        | search              | search                       |
+        | search              | se                           |
+        | shell               | shell                        |
+        | shell               | sh                           |
+        | swap                | swap                         |
+        | updateinfo          | updateinfo                   |
+        | updateinfo          | list-updateinfo              |
+        | updateinfo          | list-security                |
+        | updateinfo          | list-sec                     |
+        | updateinfo          | info-updateinfo              |
+        | updateinfo          | info-security                |
+        | updateinfo          | info-sec                     |
+        | updateinfo          | summary-updateinfo           |
+        | upgrade             | upgrade                      |
+        | upgrade             | update                       |
+        | upgrade             | upgrade-to                   |
+        | upgrade             | update-to                    |
+        | upgrade             | localupdate                  |
+        | upgrade             | up                           |
+        | upgrade-minimal     | upgrade-minimal              |
+        | upgrade-minimal     | update-minimal               |
+        | upgrade-minimal     | up-min                       |
```

Comment:
I'm wondering about the purpose and usefulness of this test though... We will also have to maintain this list, mainly not forget to add new commands here. Which is surely bound to happen...

Reply:
It's one of tests for [bug 1690289](https://bugzilla.redhat.com/show_bug.cgi?id=1690289):
https://github.com/rpm-software-management/dnf/commit/da52d0ee8ec53fc67477a448ef9da73e32ce2cf2
  MED - addition of multiple command aliases, we should create test to prevent future regressions

I am also not sure how likely it's to happen that some alias disappears by accident and the tests catch that.

Reply:
Yeah, sure, I suppose this does the job.